Gruntfile.js

```diff
@@ -130,91 +111,17 @@ module.exports = function( grunt ) {
           src: [
             "public/editor/**/*.js",
             "public/homepage/**/*.js",
+            "public/resources/remix/index.js",
```

Comment:
Should we be jshinting the resources at all? Theoretically, any web resource can be put into this folder (which may have lint errors) so should we bother linting any of them at all?